src/main/java/ladder/domain/LadderGame.java

```diff
@@ -0,0 +1,32 @@
+package ladder.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LadderGame {
+
+    private GameInfo gameInfo;
+    private Lines lines;
+
+    public LadderGame(GameInfo gameInfo, Lines lines) {
+        this.gameInfo = gameInfo;
+        this.lines = lines;
+    }
+
+    public LadderResult createResult() {
+        Map<Integer, Integer> positionGoals = new HashMap<>();
+        for (int i = 0; i < gameInfo.getLadderWidth(); i++) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요? `gameInfo.getLadderWidth()`가 n 번 호출되는 것을 막을 수 있어요.

```suggestion
        int ladderWidth = gameInfo.getLadderWidth();
        for (int i = 0; i < ladderWidth; i++) {
```